src/StoreKeeper/WooCommerce/B2C/Commands/CommandRunner.php

```diff
@@ -267,9 +267,15 @@ function ($type, $buffer) use ($context) {
                     $this->logger->error($buffer, $context);
                 } else {
                     $this->logger->debug($buffer, $context);
+                    if (!empty($buffer)) {
+                        echo $buffer;
+                    }
                 }
+                flush();
+                ob_flush();
```

Comment:
```suggestion
if( !$echoOutput) {
                    $this->logger->error($buffer, $context);
                } else {
                    $this->logger->debug($buffer, $context);
                }
} else {
 echo $buffer;
                flush();
                ob_flush();
                }
```

Maybe it should some option to the function? I looks little strange that w both log it and echo in the same time